lib/rucio/db/sqla/migrate_repo/ddl_helpers.py

```diff
@@ -0,0 +1,1552 @@
+    Remove a deprecated column from ``requests``:
+
+    >>> drop_column("requests", "old_col")
+    """
+    kwargs = _with_default_schema(dict(kwargs))
+    return op.drop_column(*args, **kwargs)
+
+
+def drop_constraint(*args: 'Any', **kwargs: 'Any') -> None:
+    """
+    Replacement for :func:`alembic.op.drop_constraint` with schema defaulting.
+
+    Parameters
+    ----------
+    *args : Any
+        Positional arguments forwarded to Alembic.
+    **kwargs : Any
+        Keyword arguments forwarded to Alembic. If ``schema`` is not provided or is
+        falsy, :func:`get_effective_schema` is injected.
+
+    Returns
+    -------
+    None
+
+    Notes
+    -----
+    This is a thin schema-defaulting wrapper. For tolerant "drop if exists" semantics,
+    use :func:`try_drop_constraint`.
+    """
+    kwargs = _with_default_schema(dict(kwargs))
+    return op.drop_constraint(*args, **kwargs)
+
+
+def drop_index(*args: 'Any', **kwargs: 'Any') -> None:
+    """
+    Replacement for :func:`alembic.op.drop_index` with schema defaulting.
+
+    Parameters
+    ----------
+    *args : Any
+        Positional arguments forwarded to Alembic.
+    **kwargs : Any
+        Keyword arguments forwarded to Alembic. If ``schema`` is not provided or is
+        falsy, :func:`get_effective_schema` is injected.
+
+    Returns
+    -------
+    None
+
+    Notes
+    -----
+    This wrapper does not swallow "missing index" errors. For idempotent semantics across
+    backends, prefer :func:`try_drop_index`.
+
+    Examples
+    --------
+    >>> drop_index("IX_ACCOUNTS_EMAIL", table_name="accounts")
+    """
+    kwargs = _with_default_schema(dict(kwargs))
+    return op.drop_index(*args, **kwargs)
+
+
+def drop_table(*args: 'Any', **kwargs: 'Any') -> None:
+    """
+    Replacement for :func:`alembic.op.drop_table` with schema defaulting.
+
+    Parameters
+    ----------
+    *args : Any
+        Positional arguments forwarded to Alembic.
+    **kwargs : Any
+        Keyword arguments forwarded to Alembic. If ``schema`` is not provided or is
+        falsy, :func:`get_effective_schema` is injected.
+
+    Returns
+    -------
+    None
+
+    Examples
+    --------
+    >>> drop_table("tmp_account_audit")
+    """
+    kwargs = _with_default_schema(dict(kwargs))
+    return op.drop_table(*args, **kwargs)
+
+
+def rename_table(*args: 'Any', **kwargs: 'Any') -> None:
+    """
+    Replacement for :func:`alembic.op.rename_table` with schema defaulting.
+
+    Parameters
+    ----------
+    *args : Any
+        Positional arguments forwarded to Alembic.
+    **kwargs : Any
+        Keyword arguments forwarded to Alembic. If ``schema`` is not provided or is
+        falsy, :func:`get_effective_schema` is injected.
+
+    Returns
+    -------
+    None
+
+    Examples
+    --------
+    >>> rename_table("collections", "collections_legacy")
+    """
+    kwargs = _with_default_schema(dict(kwargs))
+    return op.rename_table(*args, **kwargs)
+
+
+__all__ = [
```

Comment:
Worth trying to import from this file with absolute import to avoid having `__all__` both here and in the `__init__` file